RevPi-HAT-EEPROM-Format.md

```diff
@@ -236,50 +186,40 @@ The GPIO map can be used to configure GPIOs before the kernel is booted.
 | :warning: Some properties like the drive strength, slew rate and hysteresis will be applied for the whole bank (GPIOs 0-27). |
 |------------------------------------------------------------------------------------------------------------------------------|
 
+See [GPIO map atom data (type=0x0002):](#gpio-map-atom-data-type0x0002) of the [The Raspberry Pi HAT ID EEPROM FORMAT SPECIFICATION](#the-raspberry-pi-hat-id-eeprom-format-specification) for more details.
+
 ### Linux Device Tree (Blob) Atom
 
-The Linux device tree blob atom can contain a compiled device tree overlay blob (not used for Revolution Pi). Or a file with which contains the name of an overlay in ASCII format. Every Revolution Pi device has it’s own device tree overlay. This atom contains the name of the overlay.
+The Linux device tree blob atom can contain a compiled device tree overlay blob (not used for Revolution Pi). Or a file which contains the name of an overlay in ASCII format. Every Revolution Pi device has it’s own device tree overlay. This atom contains the name of the overlay.
 
 ##### Example(s) <!-- omit in toc -->
 For the Revolution Pi Connect (the original Connect, not any variant) the string is:
 `revpi-connect`
```

Comment:
Does a list of allowed values exist somewhere in this document or is it implicit knowledge of the RevPi team or people who ar willing to dig into the device, debian package or other sources?

Or simpler, is it relevant to know all allowed values/strings?

Reply:
The format specification is the wrong place for these, as we would need to update the document for every new device. The example given is just for a better understanding. Any user can create his own dt overlay and thus need to provide the matching dtstr.